src/tap/src/tapcell.tcl

```diff
@@ -1087,5 +1107,18 @@ proc remove_cells {prefix} {
   return $removed
 }
 
+proc check_symmetry {master ori} {
+  set symmetry_x [$master getSymmetryX]
+  set symmetry_y [$master getSymmetryY]
+
+  set symmetry_ok 1
+  if {(!$symmetry_x && $ori == "MX") || \
+      (!$symmetry_y && $ori == "R0")} {
+    set symmetry_ok 0
+  }
```

Comment:
I think it is better to say what is allowed rather than what isn't, as this list is incomplete.  From the LEF manual:

Possible orientations include:
X N and FS orientations should be tried.
Y N and FN orientations should be tried.
X Y N, FN, FS, and S orientations should all be tried.